PatientRegistrationApp/src/components/SinglePatientPersonalInfo.vue

```diff
@@ -158,7 +159,8 @@ export default {
       givenName: "",
       middleName: "",
       suffix: "",
-      date: "",
+      date: new Date().toISOString().substr(0, 10),
```

Comment:
I think it would be better to leave the value empty, so that it prevents the user from continuing if the field is ignored.